docs/src/main/paradox/scala/http/routing-dsl/path-matchers.md

```diff
@@ -1,3 +1,10 @@
+@@@ div { .group-java }
+
+This DSL is only relevant when using the Scala API.
```

Comment:
Should we then hide it completely maybe?

Reply:
The question is what should happen when you are on the Scala version of this page and switch to the Java version.

We could go fancy and use page attributes to mark the whole page as scala-only, and then generate a page that does a redirect (though to where?) for when you hit the the java version (https://github.com/lightbend/paradox/pull/169 might be an inspiration)... but that seems kind of tricky, this should be nice and simple for now :).